### DISCUSSION TITLE
Problems with sessions in Wiki.js 2.5.85

### DISCUSSION THREAD

**Discussion by gsanzmon** [requarks/wiki#5468] [Error / Bug Report] [answered]
**Hello , I have tried to install wiki js in Ubuntu with this manual and I have the following problems:
1) Problem closing sessions . I mean , when I log in the web with a user, I enter and see all the information. When I log out I go to the input menu. If I close the window and open the window again , the session has not been closed. Any Idea??
2) The server takes a lot of time in making a response**

**Comment by NGPixel** [comment, marked as answer]
That's not how Wiki.js sessions work. Upon login, you get a token (saved to cookies) which is valid for 30 minutes and can be renewed for 14 days. Whether you close / reopen windows is irrelevant.

You can change the timeframe for both values under Administration > Security.

**Reply by stephane873** [reply]
I am running the latest version of Wiki.js and this is still the same. The Log out function keeps the browser on the same page instead of bringing up the login page. Additionally, now any page or menu that was blocked for that user when logged in is now accessible. Pressing the login button sends to the login page but pressing "go back" brings back to the full wiki.js website with not restrictions. Closing the page and re-opening it in the history again give full site access to an un-logged user. I have changed the token timeout function of both at 1m and it makes things even worse because what I described above  is now happening after 1 minute without trying to log out. This makes this application unsecure to use on a public computer or computer used as a common work station in an office or shop. It also prevents using wiki.js if some of your customers are competitors and are only allowed to navigate certain pages per group policy. Why implement 2FA authentication if security can be bypassed so easily? Don't get me wrong, I think you have made fantastic work but unfortunately, I cannot use the Wiki in these circumstances.
